src/test/SigninTest.php

```diff
@@ -40,6 +42,12 @@ public function findById(UserId $id): ?User
                 {
                     return null;
                 }
+            },
+            new class implements AuthStateRepositoryInterface
+            {
+                public function saveAuthUser(AuthUser $authUser): void
+                {
+                }
```

Comment:
以下のようにして、 `saveAuthUser` メソッドがちゃんと呼び出されたかのテストも書いちゃいましょうか！！
接種予約のデモアプリのテストをご参考いただければと思います！😄

```suggestion
            },
            new class implements AuthStateRepositoryInterface
            {
                public ?AuthUser $authUser = null;
                
                public function saveAuthUser(AuthUser $authUser): void
                {
                    $this->authUser = $authUser;
                }
```

Reply:
レビューありがとうございました！
こちらのテストケース追加しました。

https://github.com/tech-quest/tq-twitter/pull/150/commits/04a52e19701213214bf6f90eba0e8ecbdb4f3523